assembl/static2/js/app/components/common/textInputWithRemainingChars.jsx

```diff
@@ -21,6 +22,7 @@ export const TextInputWithRemainingChars = ({
         maxLength={maxLength}
         value={value}
         onFocus={handleInputFocus || null}
+        onBlur={handleInputBlur || null}
```

Comment:
We should'nt have those `||null` for onFocus and onBlur but defined a defaultPropos `()=>{}` in the component.

Reply:
I would prefer adding the onBlur (respectively onFocus) attribute only if handleInputBlur is a defined function

Reply:
As you wish, fine by me.